services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/RateDetailsSummarySection.tsx

```diff
@@ -35,6 +35,40 @@ export const RateDetailsSummarySection = ({
         doc.documentCategories.includes('RATES_RELATED')
     )
 
+    let rateName: string
+
+    const generateRateName = (
```

Comment:
We’re you talkinga about putting this in common-code? Maybe that’s a follow on PR with the emails I’m remembering?

Reply:
Yes, this will be done in the rate name in email PR.